frontend/src/components/pages/admin/UsersPage.tsx

```diff
@@ -131,19 +123,12 @@ const UsersPage = (): React.ReactElement => {
               <TabPanel>
                 <VStack pt={4} spacing={6}>
                   <HStack width="100%">
-                    <InputGroup width="95%">
-                      <Input
-                        borderRadius="6px"
-                        borderColor="grey.100"
-                        backgroundColor="grey.100"
-                        onChange={(e) => setSearch(e.target.value)}
-                        placeholder="Search bar"
-                      />
-                      <InputRightElement pointerEvents="none" h="full">
-                        <SearchOutlineIcon />
-                      </InputRightElement>
-                    </InputGroup>
-                    <SortTablePopover OrderingSets={OrderingSets} />
+                    <SearchBar onSearch={setSearch} />
+                    <SortMenu
+                      tableProperties={["firstName", "email"]}
```

Comment:
is there a reason we pass the values like so? rather than `["Name", 
"Email"]  - these will be consistent with both the table headers and sort properties

then we can avoid the workarounds in line 45 of `SortMenu`

Reply:
It's because `adminUser` has `firstName`, `lastName` and `email` as fields, so you can have to sort by one of those. I think in the (hopefully) last refactor for the `UsersPage` I would concatenate the first two fields and have the `AdminUser` data stored in an object that just has a Name` and `Email` field. Let me know if that makes sense!

Reply:
ohh makes sense !